miscellaneous/glossary.md

```diff
@@ -0,0 +1,89 @@
+# Glossary
+
+This glossary is a collection of terms and definitions used throughout the Safe{Core} documentation.
+
+## A
+
+### Account
+
+An account is a user's Ethereum account. It is represented by an Ethereum address and is used to sign transactions and messages. It can also be found as a EOA (Externally Owned Account). The main vision of the Safe{Core} is to provide a safe and secure way to improve Ethereum accounts, that we refer to as [Smart Accounts](#smart-accounts).
+
+### Account Abstraction
+
+Account abstraction is a term used to describe the process of abstracting the concept of an account. It is a way to improve the security of Ethereum accounts by adding a layer of abstraction on top of them, adding features for [recoverability](#recoverability), [social logins](#social-login) and [gasless transactions](#gasless-transactions). This layer of abstraction is the main concept we use to build Safe's vision for modular [Smart Accounts](#smart-accounts).
+
+### Apps
+
+Safe Apps are web applications that can be used to interact with Safe. They are used to provide a better user experience when interacting with Safe, and are mostly built using the [Safe Apps SDK](../safe-apps/README.md). They can be used to interact with Safe's core features, such as [transactions](#transactions) and [modules](#modules), or to provide entirely new features. They are gathered in the a [Safe App Gallery](https://app.safe.global/apps) and can be used by any Safe{Wallet} user.
+
+### AuthKit
+
+AuthKit is a part of the Safe{Core} SDK that provides a set of tools to authenticate users and sign messages. It is used to sign transactions and messages with the user's Ethereum account. It also provides a set of tools to authenticate users with more familiar web services, such as Gmail. In this last case, we talk about [Social Login](#social-login).
```

Comment:
```suggestion
The AuthKit is a library of the Safe{Core} SDK that provides a set of tools to authenticate users with one of the different [Social Login](#social-login) methods offered, generating an externally-owned account that can act as the signer of a Safe.
```

For reference: https://github.com/safe-global/safe-core-sdk/tree/main/packages/auth-kit